ui/app/components/namespace-link.js

```diff
@@ -30,4 +31,23 @@ export default Component.extend({
   isCurrentNamespace: computed('targetNamespace', 'currentNamespace', function() {
     return this.currentNamespace === this.targetNamespace;
   }),
+
+  get namespaceLink() {
+    if (Ember.testing) {
+      if (this.normalizedNamespace) {
+        return `/ui/vault/secrets?namespace=${this.normalizedNamespace}`;
+      }
+      return `/ui/vault/secrets`;
+    }
+
+    let origin =
+      window.location.protocol +
+      '//' +
+      window.location.hostname +
+      (window.location.port ? ':' + window.location.port : '');
+
+    if (!this.normalizedNamespace) return `${origin}/ui/vault/secrets`;
+
+    return `${origin}/ui/vault/secrets?namespace=${encodeURIComponent(this.normalizedNamespace)}`;
```

Comment:
Maybe add a note here that the full URL/origin is required so that the page is reloaded?